### ISSUE TITLE
In certain (to be determined) circumstances, 'NaN' is sent in a value for the pp_mix or pp_max values

### ISSUE THREAD

**Issue by yalisassoon** [snowplow/snowplow-javascript-tracker#324]
Example error row:

```json
{
    "line": "2015-02-18\t11:02:51\t-\t43\t195.43.48.140\tGET\t195.43.48.140\t/i\t200\thttp://www.blahblah.com/en/uk/flowers/same-day-delivery\tMozilla%2F4.0+%28compatible%3B+MSIE+8.0%3B+Windows+NT+6.1%3B+Trident%2F4.0%3B+SLCC2%3B+.NET+CLR+2.0.50727%3B+.NET+CLR+3.5.30729%3B+.NET+CLR+3.0.30729%3B+.NET+CLR+1.1.4322%3B+.NET4.0C%3B+.NET4.0E%3B+InfoPath.3%29\te=pp&url=http%3A%2F%2Fwww.blahblah.com%2Fen%2Fuk%2Fflowers%2Fsame-day-delivery&page=Same-Day%20Flowers%20Delivered&refr=http%3A%2F%2Fwww.blahblah.com%2F%3Fcatid%3D2241%26kwnm%3D%252bsame%2B%252bday%2B%252broses%26channel%3Dppc%26gclid%3Dcky-mckr68mcfqoxwwodzsoapq&pp_mix=NaN&pp_max=NaN&pp_miy=1562&pp_may=1562&tv=js-2.2.0&tna=co&aid=blahblah.com&p=web&tz=Europe%2FLondon&lang=en-gb&cs=utf-8&f_java=1&res=1600x900&cd=32&cookie=1&eid=1a2b70a9-baa8-491e-9c06-787884eb99ab&dtm=1424257395429&vp=1579x694&ds=1600x3463&vid=1&duid=0206beae38854bcf&fp=1758745377&cx=eyJzY2hlbWEiOiJpZ2x1OmNvbS5zbm93cGxvd2FuYWx5dGljcy5zbm93cGxvdy9jb250ZXh0cy9qc29uc2NoZW1hLzEtMC0wIiwiZGF0YSI6W3siZGF0YSI6eyJzY2hlbWEiOiJpZ2x1OmNvbS5zZXJlbmF0YWZsb3dlcnMvcGFnZS9qc29uc2NoZW1hLzEtMC0wIiwibmFtZSI6IlNhbWUtRGF5IEZsb3dlcnMgRGVsaXZlcmVkIiwidXJsUGF0aCI6Ii9lbi91ay9mbG93ZXJzL3NhbWUtZGF5LWRlbGl2ZXJ5In19XX0&cv=clj-0.9.1-tom-0.1.0&nuid=973b7b73-b757-4a2a-942c-a2265c944d82\t-\t-\t-\t-\t-",
    "errors": [
        "Field [pp_mix]: cannot convert [NaN] to Int",
        "Field [pp_max]: cannot convert [NaN] to Int"
    ]
}
```

**Comment by yalisassoon** [comment]
Note that this occurs with `[pp_may]` and '[pp_miy]` values as well

**Comment by yalisassoon** [comment]
Some example useragent strings where this occurs:

* Mozilla%2F4.0+%28compatible%3B+MSIE+8.0%3B+Windows+NT+6.1%3B+Trident%2F4.0%3B+SLCC2%3B+.NET+CLR+2.0.50727%3B+.NET+CLR+3.5.30729%3B+.NET+CLR+3.0.30729%3B+.NET+CLR+1.1.4322%3B+.NET4.0C%3B+.NET4.0E%3B+InfoPath.3%29
* Mozilla%2F4.0+%28compatible%3B+MSIE+7.0%3B+Windows+NT+6.1%3B+Trident%2F4.0%3B+FBSMTWB%3B+SLCC2%3B+.NET+CLR+2.0.50727%3B+.NET+CLR+3.5.30729%3B+.NET+CLR+3.0.30729%3B+Media+Center+PC+6.0%3B+SLCC1%3B+InfoPath.3%3B+Alexa+Toolbar%29

Looks like an issue for IE 7 and IE 8 users (there are still some!)...

**Comment by fblundun** [comment]
Current hypothesis: for those browsers the getPageOffsets function returns undefined. The offsets used to be simply omitted from the querystring, but the fix for #291 converted the undefined values to NaNs, which are (currently) allowed into the querystring.
